pkgs/sagemath-objects/tox.ini

```diff
@@ -89,14 +89,15 @@ package_env = .pkg-sagepython
 basepython = {env:SAGE_VENV}/bin/python3
 package_env = .pkg-sagepython-sagewheels-nopypi
 
-[testenv:sagepython-sagewheels-nopypi-norequirements]
-basepython = {env:SAGE_VENV}/bin/python3
-package_env = .pkg-sagepython-sagewheels-nopypi
-
 [testenv:sagepython-sagewheels]
 basepython = {env:SAGE_VENV}/bin/python
 package_env = .pkg-sagepython
 
 [testenv:sagepython-norequirements]
 basepython = {env:SAGE_VENV}/bin/python3
 package_env = .pkg-sagepython
+
+
+[testenv:sagepython-sagewheels-nopypi-norequirements]
```

Comment:
Two lines, intended?

Reply:
Yes, this separates the default environment from the others above

Reply:
OK.